core/src/main/scala/cmt/CMTcConfig.scala

```diff
@@ -22,7 +22,7 @@ class CMTcConfig(studentifiedRepo: File):
   import Helpers.adaptToOSSeparatorChar
 
   private val cmtConfigFile = studentifiedRepo / ".cmt/.cmt-config"
-  if !cmtConfigFile.exists then printErrorAndExit("missing CMT configuration file")
+  if !cmtConfigFile.exists then printErrorAndExit("Missing CMT configuration file, have you set your current course?")
```

Comment:
Maybe the message should be tied a bit more. The _"Missing CMT configuration file"_ bit is not helpful to students as it points to an implementation detail. Perhaps it could be something along the lines of:

"It seems no current course has been set. Use the `cmtc set-current-course` command to set one"